config/locales/en.yml

```diff
@@ -1808,6 +1808,7 @@ en:
     new:
       title: "Login"
       tab_title: "Login"
+      login_to_authorize_html: "You need an OpenStreeMap account to access %{client_app_name}."
```

Comment:
Should this be more like `Login to OpenStreetMap to access...` maybe? At the very least it needs to spell OpenStreetMap correctly ;-)